xsd/netex_part_3/part3_fares/netex_usageParameterEligibility_version.xsd

```diff
@@ -464,7 +464,19 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Parent USER PROFILE or GROUP TICKET for which this  specifes the member rules.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element ref="UserProfileRef" minOccurs="0"/>
+			<xsd:choice>
+				<xsd:element ref="UserProfileRef" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>DEPRECATED: use list of userProfiles - kept for backwards compatibility</xsd:documentation>
```

Comment:
Would it make sense to add a version note here, too? "-V2.0" indicating that it is not an addition but a restriction?

Reply:
@Aurige  What do you think?

Reply:
fine for me .. additional explanations are always good ;-) 
But the UserProfileRef was there previously, the 2.0 addition is **userProfiles**